elasticapm/conf/__init__.py

```diff
@@ -66,6 +66,81 @@ def __init__(self, msg, field_name):
         super(ValueError, self).__init__(msg)
 
 
+class Seconds:
```

Comment:
That was my first implementation. The problem is how Python handles cases where a [comparison method](https://docs.python.org/3/reference/datamodel.html#object.__lt__) returns `NotImplemented`. In such a case, it seems to try the reflected method (e.g. `__lt__` vs. `__gt__`) on the other object. E.g. if we have

    `a_seconds_object < 1.0`

`a_seconds_object.__lt__(1.0)` returns `NotImplemented, so Python tries `1.0.__gt__(a_seconds_object)`, which would work if `Seconds` is a subclass of `float`.

Implementing `Seconds` by encapsulating instead of inheriting from `float` lets us define the behaviour exactly as we want it, with the drawback that we need to implement all the methods ([`functools.total_ordering`](https://docs.python.org/3/library/functools.html#functools.total_ordering) could help with that, though).

Reply:
Cool, sounds fine. :)